oml/configs/criterion/arcface.yaml

```diff
@@ -0,0 +1,7 @@
+name: arcface
+args:
+  smoothing_epsilon: 0
```

Comment:
let's replace `0` with `0.0` to explicitly show that this is a float parameter (the same for other places)